tests/e2e/objects/quickInputs/Quickpick.ts

```diff
@@ -11,6 +11,18 @@ export class Quickpick {
     return this.page.locator(".quick-input-widget");
   }
 
+  /** The quickpick header container. */
+  get header(): Locator {
+    return this.locator.locator(".quick-input-header");
+  }
+
+  /** The quickpick input field where you can type to filter items. */
+  get textInput(): Locator {
+    return this.header.locator(
+      ".quick-input-and-message .quick-input-filter .quick-input-box .monaco-findInput .monaco-inputbox .ibwrapper input.input",
+    );
+  }
```

Comment:
Might be able to simplify this; currently only used to filter commands from the palette in `executeVSCodeCommand()`